docs/index.html

```diff
@@ -32,6 +32,8 @@ <h1>Latest release (3.11.0)</h1>
       <img style="display: block; margin-left: auto; margin-right: auto;" src="img/msvc2015-dll.png" width="300px">
       <p>your machine has not had recent Windows updates applied. Please either update Windows fully or install the relevant
 	<a href="https://support.microsoft.com/en-gb/help/2999226/update-for-universal-c-runtime-in-windows">standalone</a> update package for your version of x64 Windows.</p>
+    <div class="windows">
```

Comment:
I would suggest removing this `div` and changing `windows-upgrade` above to just `windows`.